packages/select/package.json

```diff
@@ -48,11 +48,11 @@
     "@react-aria/select": "^3.10.0",
     "@react-aria/utils": "^3.16.0",
     "@react-stately/collections": "^3.7.0",
-    "@react-stately/list": "^3.7.0",
+    "@react-stately/list": "^3.8.0",
```

Comment:
Need to handle both deps upgrades for the select since they are dependent on each other